### PR TITLE
Swap request.DATA for request.data

### PR DESCRIPTION
**Pull request by adam-incuna**
@incuna/backend merge please? :)

### CODE REVIEW

**Comment by adam-incuna** [comment]
Wait, I missed a requirements update in `setup.py`.  Commit incoming.

**Comment by adam-incuna** [comment]
Also tests are sad.  Will fix.

**Comment by adam-incuna** [comment]
@incuna/backend This is now less rushed.  Review/merge please? :)

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 install_requires = (
-    'djangorestframework>=3.1.0,<3.2',
+    'djangorestframework>=3.2.0,<3.3',
```
Comment (Ian-Foote) [inline setup.py]: Do we need to drop `v3.1` support?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## Upcoming (v13.1.1)
+
+* Swap `request.DATA` (deprecated in DRF v3.0, removed in DRF v3.2) for `request.data`.
```
Comment (meshy) [inline CHANGELOG.md]: If we're dropping DRF 3.1, this is backwards incompatible.

Comment (adam-incuna) [inline CHANGELOG.md]: I don't think we are, though.

Comment (adam-incuna) [inline CHANGELOG.md]: Unless that's what the setup.py change implies.  I see. Um

File: setup.py (hunk 1 above)
Comment (adam-incuna) [inline setup.py]: Not that I know of, I just wanted it to use 3.2 for tests/local installs.  Do I have to faff around to maintain 3.1 support as well?

Comment (adam-incuna) [inline setup.py]: Or will this command fetch the newest possible version of DRF?

Comment (adam-incuna) [inline setup.py]: Yeah it does, doesn't it.  I'll change it to >3.1.0.

Comment (Ian-Foote) [inline setup.py]: Only if you've broken compatibility. If `3.1` still works, just change the `>=3.2.0,<3.3` to `>=3.1.0,<3.3`.

We could also include the DRF version in the tox setup for travis. That would double the test runs though...

File: CHANGELOG.md (hunk 2 above)
Comment (adam-incuna) [inline CHANGELOG.md]: No longer inadvertently dropping 3.1!

**Comment by adam-incuna** [comment]
Updated!

**Comment by adam-incuna** [comment]
Bump?  This is a bit urgent.

**Comment by adam-incuna** [comment]
Thanks! :)

**Pull request merged**